model_analyzer/record/metrics_manager.py

```diff
@@ -130,35 +136,47 @@ def profile_model(self, perf_config, perf_output_writer=None):
         model_non_gpu_metrics = list(perf_analyzer_metrics.values()) + list(
             model_cpu_metrics.values())
 
-        return model_gpu_metrics, model_non_gpu_metrics
+        measurement = None
+        if model_gpu_metrics is not None and model_non_gpu_metrics is not None:
```

Comment:
This used to be here, but then we moved it to `_execute_run_config` for the run config search PR. Wondering what made you move it back here? I agree it looks cleaner.

Reply:
Just wanted to make sure that it returns a measurement similar to `profile_server`.